docs/use-cases/Retirement.md

```diff
@@ -0,0 +1,246 @@
+
+Retirement Providers will have their own screens for sign up and enrollment of employees of the company, but will require that information be transferred back to QBO Payroll so that the correct deductions can be taken. 
+ 
+They will then need to write (and read) these fields for contribution per employee level. The only person who can edit the names or fields of each contribution will be the creator. If you are taking over from a different company as Retirement provider you will need to work with QBO company's payroll Admins to sunset contributions from previous plan. 
+
+* Traditional % or Traditional $
+* Roth % or Roth $
+* loan amount $ 
+* Loan ID
+* Employer contribution %
+* Employer contributions limit
+* Nonelective contribution 
+* Deduction Code
+
+Setup up a deduction for the employer 
+````
+mutation {
+ 
+  createEmployerDeduction (
+    
+    deduction : {
+      name : "My plan deduction"
+      statutoryDeductionPolicy: "Annual max is 10k"
+    }
+  ) {
+    id
+    name
+  }
+}
+````
+
+For each employee on the plan, get their ID using the roster query and setup a deduction.
+
+```
+mutation {
+  
+  createEmployeeDeduction (
+    deduction : {
+      employeeId: "<employee id>"
+      active: true
+      amount: {
+        value : 500
+        percentage : false
+      }
+      employerDeductionId: "<id from employer deduction>"
+    } 
+  ) {
+    id
+    active
+    amount {
+      percentage
+      value
+    }
+  }
+}
+```
+
+#### Paycheck Information
+
+Retirement providers need to download Payslips after each payroll run so they know how much in deductions to take out for the employees, and this is where deductions details are most important.   At the beginning of the plan, providers will need to query for up to all of last fiscal year’s Payslips to determine plan eligibility for each employee:
+
+* Pay date
+* Payroll Process Date (approved date)
+* Employee ID
+* SSN (again above entitlements)
+* Paycheck ID
+* Gross Pay
+* YTD Pay (optional, not launch blocker)
+* Payroll Type (optional - bonus, overtime, regular, etc)
+* Hours Worked
+* Amount deducted for each category (employee)
+  * Roth
+  * Traditional
+  * Loan
+* Employer contribution  
+
+After establishing connection with QBO company run this  query to download Payslips periodically.
+
+```
+query readAllEmployeePayslips {
+  company {
+	id
+    employees{
+      nodes {
+        id
+        }
+        payslips {
+            id
+            payDate
+            grossPay {
+              amount
+              yearToDateAmount
+            }
+            contributions {
+              description
+              contributionAmount {
+                amount
+                yearToDateAmount
+              }
+            }
+            deductions {
+              type
+              description
+              deductionAmount {
+                amount
+                yearToDateAmount
+              }
+            }
+          }
+      }
+    }
+  }
+}
```

Comment:
There was an extra `}` in the query that was breaking the query.  Minor spacing fix as well

```suggestion
query readAllEmployeePayslips {
  company {
    id
    employees{
      nodes {
        id
        payslips {
          id
          payDate
          grossPay {
            amount
            yearToDateAmount
          }
          contributions {
            description
            contributionAmount {
              amount
              yearToDateAmount
            }
          }
          deductions {
            type
            description
            deductionAmount {
              amount
              yearToDateAmount
            }
          }
        }
      }
    }
  }
}
```